src/plugins/presentation_util/public/components/expression_input/expression_input.tsx

```diff
@@ -75,7 +75,7 @@ export const ExpressionInput = (props: ExpressionInputProps) => {
 
   return (
     <div style={{ height, ...style }} {...{ rest }}>
-      <CodeEditor
+      <CodeEditor // TODO CodeOwners will need to help with this one
```

Comment:
Expose getTheme method in https://github.com/elastic/kibana/blob/main/x-pack/plugins/file_upload/public/kibana_services.ts and then use the getter function to get theme in this component